### PR TITLE
Update 08-divyadaram.md

### PR DESCRIPTION
**Pull request by divyadaram**
Changed text and added caption

### CODE REVIEW

**Review by divyadaram — COMMENTED** [review]
sample

**Comment by mcbanks12** [comment]
Hello and passed 👍

File: _slides/08-divyadaram.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Smart cat learning github.
```
Comment (mcbanks12) [inline _slides/08-divyadaram.md:6]: Hope I as Smart

**Review by mcbanks12 — APPROVED** [review]
Looks good

**Pull request merged**